index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="shortcut icon" type="image/png" href=".\/images/light.png" />
+		<link rel="stylesheet" href="css/style.css" />
+		<link rel="stylesheet" href="css/fontawesome.css" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;1,300;1,400&display=swap"
+			rel="stylesheet"
+		/>
+		<title>Ken Sparks | To Do List</title>
+	</head>
+	<body>
+		<main>
+			<div class="content">
+				<p class="todays-motivation">Today's Motivation</p>
+				<div class="random-quote">
+					<div class="random-quote-text">
+						<p class="quote"><i>What is grief if not love persevering?</i></p>
```

Comment:
Don't use `<i>` in your code. Use CSS to achieve italics. The `<i>` should be reserved by WYSIWYG editors that are forced to insert those tags.